packages/ast-spec/src/declaration/ExportAllDeclaration/fixtures/type-kind/snapshots/5-AST-Alignment-AST.shot

```diff
@@ -0,0 +1,44 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`AST Fixtures declaration ExportAllDeclaration type-kind AST Alignment - AST 1`] = `
+"Snapshot Diff:
+- TSESTree
++ Babel
+
+  Program {
+    type: 'Program',
+    body: Array [
+      ExportAllDeclaration {
+        type: 'ExportAllDeclaration',
+        assertions: Array [],
+        exported: null,
+        exportKind: 'type',
+        source: Literal {
+          type: 'Literal',
+          raw: '\\\\'a\\\\'',
+          value: 'a',
+
+          range: [19, 22],
+          loc: {
+            start: { column: 19, line: 1 },
+            end: { column: 22, line: 1 },
+          },
+        },
+
+        range: [0, 23],
+        loc: {
+          start: { column: 0, line: 1 },
+          end: { column: 23, line: 1 },
+        },
+      },
+    ],
+-   sourceType: 'module',
++   sourceType: 'script',
```

Comment:
Wow really - babel doesn't think that `export type * from 'a';` should be considered a module? That's surprising to me!